src/main/java/com/cburch/logisim/fpga/data/IOComponentTypes.java

```diff
@@ -66,6 +66,10 @@
    * Open and constant are used in the map dialog to be able to map IOcomponents
    * to a constant or a open (hence no connection external of the FPGA/CPLD).
    */
+  
+  public static final int rotationZero = 0;
+  public static final int rotationMinusNinety = -90;
```

Comment:
Also, `Plus` and `Minus` are relative so tells nothing about the direction. Maybe commonly used `CW` and `CCW` instead?

Reply:
CW and CCW are also relative, and do not tell the angle, hence I think that -90 and +90 are clearer. Maybe -90 could be transformed in 270, but I think that +/- 90 is the most clear definition.

Reply:
I meand `CW/`CCW` combined with `90`, but maybe using just `Right` and `Left` would be clearer i.e. `rotationRight90`?